app/routes/circulars/index.tsx

```diff
@@ -42,6 +41,11 @@ export async function loader({ request: { url } }: DataFunctionArgs) {
   const startDate = searchParams.get('startDate') || undefined
   const endDate = searchParams.get('endDate') || undefined
   const page = parseInt(searchParams.get('page') || '1')
+  const limitRequest = parseInt(searchParams.get('limit') || '100')
+  var limit = 100
+  if (limitRequest < 100) {
+    limit = limitRequest
+  }
```

Comment:
Try [`clamp` from lodash](https://lodash.com/docs/4.17.15#clamp) here.

Reply:
oh, that's much cleaner, thanks for the tip